src/service/real-time-config/real-time-config-impl.js

```diff
@@ -134,7 +130,11 @@ export class RealTimeConfigManager {
    * @param {string} errorReportingUrl
    */
   sendErrorMessage(errorType, errorReportingUrl) {
-    if (!ERROR_REPORTING_ENABLED) {
+    if (
+      !getMode(this.win_).localDev &&
+      !getMode(this.win_).test &&
+      Math.random() >= 0.01
```

Comment:
will this spike error reporting? i.e. instead of once per document we are doing this random check on every call?

Reply:
This is the current behavior, we are already sampling the log:

https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/real-time-config-manager.js#L36